api/balances.js

```diff
@@ -28,67 +29,49 @@ const DefaultPageLimit = 200;
  *  @param {Number String} [request.query.ledger] - identifier
```

Comment:
Yes, but I didn't want to update them until everyone agrees on the calling syntax.